### DISCUSSION TITLE
Purpose of script created by Quickemu in the VM configuration directory?

### DISCUSSION THREAD

**Discussion by JAAlperin** [quickemu-project/quickemu#1248] [Quickemu]
My macos-sonoma directory contains macos-sonoma.sh.  If I enable and execute it, nothing seems to happen.  What is it's purpose?

**Comment by lj3954** [comment]
The script should launch QEMU in exactly the same way quickemu last did.

**Comment by JAAlperin** [comment]
Unfortunately, the script does not launch the vm, and no log entry is created.

**Reply by lj3954** [reply]
Please provide the contents of the script.

**Comment by TuxVinyards** [comment]
This file causes continual confusion.  It's just a record of the command that Quickemu sends to Qemu.

Much of Quickemu is currently under revision and documentation is improving.  I set up the [qqX](https://github.com/TuxVinyards/qqX) project independently so I could add things that I felt were needed but without waiting months for them to be addressed or even dismissed.

This is what qqX adds to the top of that file:

```
#!/usr/bin/env bash

#  qqX:  This file is NOT used by quickemu to run the VM but could run it if the script is placed

#  in the calling folder, where the .conf file is located, and if it is given execute permission

#  You may need to cross check with qqX Args file as qqX may have issued a modified version ... 

/usr/bin/qemu-system-x86_64 \
    -name macos-sonoma,process=macos-sonoma \
    -pidfile macos-sonoma/macos-sonoma.pid \

```

**Comment by flexiondotorg** [comment]
@TuxVinyards The self-promotion of your project while also making disrespectful comments about Quickemu and me is unbecoming. Stop it.

**Comment by TuxVinyards** [comment]
@flexiondotorg  The huge amount of work that you have put into your project is _much_ appreciated.   

I was surprised to see your comment and I am  sorry if my attempts to help came over, or if any come over, as disrespectful.

Indisputably, both the Quickemu documentation and the Quickemu code have improved greatly during the last two months. 

I know you are working hard and want to get things right.  Please don't "kill the messenger"  if I point out a little areas where further improvements could be made. I just want to see the Quickemu project made better still. That's all.  Or if I make what I think are little improvements by myself to the code that I look after.  

The reference to qqX was only to place context. Nothing more. 

I look forward to us finding a more even keel.
